### PR TITLE
docs: Explain why two imports are needed in Dagger CUE API

### PR DESCRIPTION
**Pull request by gerhard**
Used standard types & low-level primitives for the headings too.

Moved the types defined in core out of the standard types table, so that all definitions in core are grouped under their own section.

Fixes https://github.com/dagger/dagger/issues/1903
Fixes https://github.com/dagger/dagger/issues/1931

### CODE REVIEW

File: website/src/theme/DocPage/index.js [hunk 1]
```diff
@@ -139,12 +139,12 @@ function DocPage(props) {
...
-        apiEndpoint: `${window.location.hostname}/t`
-      })
-      amplitude.getInstance().logEvent('Docs Viewed', { "hostname": window.location.hostname, "path": location.pathname });
-  }, [location.pathname])
+  // useEffect(() => {
+  //     var instance1 = amplitude.getInstance().init(process.env.REACT_APP_AMPLITUDE_ID, null, {
+  //       apiEndpoint: `${window.location.hostname}/t`
+  //     })
+  //     amplitude.getInstance().logEvent('Docs Viewed', { "hostname": window.location.hostname, "path": location.pathname });
+  // }, [location.pathname])
```
Comment (TomChv) [inline website/src/theme/DocPage/index.js:147]: Should be in a separate PR since it's a fix, that could be reverted anytime

**Review by TomChv — APPROVED** [review]
I like the inclusive writing with `we` and your Notes! It feel really clear and simple to follow.

Doc is better every day! :rocket:

File: website/src/theme/DocPage/index.js (hunk 1 above)
Comment (gerhard) [reply website/src/theme/DocPage/index.js:147]: There are multiple PRs in this one, that last commit was the biggest one by far.

I will run this by @slumbering and see how he wants to improve this specific part.

Thank you for the comments Tom! 🙌🏻

Comment (slumbering) [reply website/src/theme/DocPage/index.js:147]: Ok. What we need is a .env.local with an empty ﻿REACT_APP_AMPLITUDE_ID var.

Comment (slumbering) [reply website/src/theme/DocPage/index.js:147]: I create a PR for that

Comment (slumbering) [reply website/src/theme/DocPage/index.js:147]: PR #1941
